src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -55,6 +55,7 @@ def load_arguments(self, _):
         c.argument('account_name', completer=None)
         c.argument('key_uri', help="The URI of the key vault", is_preview=True)
         c.argument('enable_free_tier', arg_type=get_three_state_flag(), help="If enabled the account is free-tier.", is_preview=True)
+        c.argument('assign_identity', arg_type=get_three_state_flag(), help="Flag to assign system assigned identity to the account.", is_preview=True)
```

Comment:
Instead of adding an extra param, why cant we just check if default_identity is not null.